README.md

```diff
@@ -87,9 +87,25 @@ typeGuards.normandy_assertAddonRollbackArguments(userInput);
 
 ### Python
 
-Native support coming soon. For now, consider using the NPM package and loading the schemas directly
-from the filesystem. After installing this package via `npm`, the schemas will be available at
-`node_modules/@mozilla/rapid-experiments-shared/schemas/<group>/<type>.json`.
+Basic native support is available via the PyPI package `mozilla-nimbus-shared`. The only function is
+exposed is one to validate an object against a named schema.
+
+```python
+from mozilla_nimbus_shared import check_schema
```

Comment:
Sorry, I don't Python very well. Why is it `mozilla-nimbus-shared` (hyphens) above, but `mozilla_nimbus_shared` (underscores) here?

Reply:
Package names are encouraged to by kebab-case, but module names must be valid identifiers. This is a pretty common pattern in Python-land.